src/ex1/style.css

```diff
@@ -0,0 +1,172 @@
+@import url("https://fonts.googleapis.com/css2?family=Chakra+Petch:wght@600&family=Nova+Square&display=swap");
+
+:root {
+   background: linear-gradient(rgb(164, 198, 216), rgb(110, 200, 248));
+}
+
+* {
+   box-sizing: border-box;
+   margin: 0;
+   padding: 0;
+   font-size: 62.5%;
```

Comment:
I am not sure this is a valid css value for `font-size`

Reply:
I think it is sometimes used, but definitely just a nice to have option:
https://stackoverflow.com/a/43554897/18682497